src/base/net/downloadmanager.cpp

```diff
@@ -148,10 +148,20 @@ Net::DownloadManager::DownloadManager(QObject *parent)
         QStringList errorList;
         for (const QSslError &error : errors)
             errorList += error.errorString();
-        LogMsg(tr("Ignoring SSL error, URL: \"%1\", errors: \"%2\"").arg(reply->url().toString(), errorList.join(u". ")), Log::WARNING);
 
-        // Ignore all SSL errors
-        reply->ignoreSslErrors();
+        QString errorMsg;
+        if (Preferences::instance()->isIgnoreSSLErrors())
+        {
+            errorMsg = tr("SSL error, URL: \"%1\", errors: \"%2\"");
```

Comment:
What does the log look like in case of SSL error? Does it contain two messages (this and the usual message about failed request)?

Reply:
Keep in mind that I merely kept the original formatting and only deleted the "Ignoring" word.
One example from my machine:
```
27/9/2024 7:18 μμ - SSL error, URL: "https://xxxxxxx.com/favicon.ico", errors: "The host name did not match any of the valid hosts for this certificate"
```

Reply:
> Keep in mind that I merely kept the original formatting and only deleted the "Ignoring" word. One example from my machine:
> 
> ```
> 27/9/2024 7:18 μμ - SSL error, URL: "https://xxxxxxx.com/favicon.ico", errors: "The host name did not match any of the valid hosts for this certificate"
> ```

You probably didn't understand what I was talking about. I'm not talking about formatting this message. I'm asking if there is also a regular error message due to a failed request? (If you don't ignore the SSL error, it leads to a failed request, right?)

Reply:
>I'm asking if there is also a regular error message due to a failed request? (If you don't ignore the SSL error, it leads to a failed request, right?)

For the favico failure I don't get any additional error logged.
Should I investigate all users of downloadmanager?

Reply:
> For the favico failure I don't get any additional error logged.

OK, I've sorted it out.
Download manager does not produce a log message for each failed request, but leaves it to a specific caller. Therefore, I would not do it for case of SSL error either.
Note that the case when SSL error is ignored is different in that the request is not considered as failed, but the user is still notified of the ignored error.